components/dashboard/src/App.tsx

```diff
@@ -4,188 +4,191 @@
-                    <Redirect to="/account"/>
-                </Route>
-                <Route path={["/access-control"]} exact>
-                    <Redirect to="/integrations"/>
-                </Route>
-                <Route path={["/subscription", "/usage", "/upgrade-subscription"]} exact>
-                    <Redirect to="/plans"/>
-                </Route>
-                <Route path={["/admin"]} exact>
-                    <Redirect to="/admin/users"/>
-                </Route>
-                <Route path="/sorry" exact>
-                    <div className="mt-48 text-center">
-                        <h1 className="text-gray-500 text-3xl">Oh, no! Something went wrong!</h1>
-                        <p className="mt-4 text-lg text-gitpod-red">{decodeURIComponent(getURLHash())}</p>
-                    </div>
-                </Route>
-                <Route path="*" /* status={404} */>
-                    <div className="mt-48 text-center">
-                        <h1 className="text-gray-500 text-3xl">404</h1>
-                        <p className="mt-4 text-lg">Page not found.</p>
-                    </div>
-                </Route>
-            </Switch>
-        </div>
-    </Route>;
-
-    const hash = getURLHash();
-    const isCreation = window.location.pathname === '/' && hash !== '';
-    const isWsStart = /\/start\/?/.test(window.location.pathname) && hash !== '';
-    if (isWhatsNewShown) {
-        toRender = <WhatsNew visible={true} onClose={() => setWhatsNewShown(false)} />;
-    } else if (isCreation) {
-        toRender = <CreateWorkspace contextUrl={hash} />;
-    } else if (isWsStart) {
-        toRender = <StartWorkspace workspaceId={hash} />;
-    }
-
-    return (
-        <Suspense fallback={<Loading />}>
-            {toRender}
-        </Suspense>
-    );
-}
-
-function getURLHash() {
-    return window.location.hash.replace(/^[#/]+/, '');
-}
-
-const renderMenu = (user?: User) => {
-    const list = [
-        {
-            title: 'Workspaces',
-            link: '/workspaces',
-            alternatives: ['/']
-        },
-        {
-            title: 'Settings',
-            link: '/settings',
-            alternatives: settingsMenu.flatMap(e => e.link)
-        }
-    ];
-
-    if (user && user?.rolesOrPermissions?.includes('admin')) {
-        list.push({
-            title: 'Admin',
-            link: '/admin',
-            alternatives: adminMenu.flatMap(e => e.link)
-        });
-    }
-
-    list.push({
-        title: 'Docs',
-        link: 'https://www.gitpod.io/docs/',
-        alternatives: []
-    });
-    list.push({
-        title: 'Community',
-        link: 'https://community.gitpod.io/',
-        alternatives: []
-    });
-
-    return <Menu list={list}/>;
-}
-
-export default App;
+ import React, { Suspense, useContext, useEffect, useState } from 'react';
+ import Menu from './components/Menu';
+ import { BrowserRouter } from "react-router-dom";
```

Comment:
<details>
<summary>:warning: <strong>This code is using `"` double-quotes in `require` or `import` statement.</strong> (click for details)</summary>

A possible improvement would be to follow best conventions of JS and use `'` single-quote to declare path in `require` statements.  
  
This change would make the code clearer and consistent.

**[More details](https://confluence.devfactory.com/x/DXFnFQ)**

## The improvement suggestion in this case
```tsx  
import { BrowserRouter } from 'react-router-dom';
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="fc921aaf-17d0-46ee-8a8a-dafd9fed9a3f" data-finding-id="359100685" href="none" codereview-uuid-reserved-tag data-business-id="507be860-41ae-46a0-83e9-7f8e359e3039" data-analyzer="js" />)